src/components/LineChart/LineChart.jsx

```diff
@@ -0,0 +1,462 @@
+		 */
+		yAxisTitleColor: number,
+
+
+		/**
+		 * An array of your y2 axis fields. Typically this will just be a single
+		 * item unless you need to display multiple lines.
+		 */
+		y2AxisFields: arrayOf(string),
+		/**
+		 * The minimum number the y2 axis should display. Typically this should be
+		 * `0`.
+		 */
+		y2AxisMin: number,
+		/**
+		 * The maximum number the y2 axis should display. This should almost always
+		 * be the largest number from your dataset.
+		 */
+		y2AxisMax: number,
+		/**
+		 * An optional function used to format your y2 axis data. If you don't
+		 * provide anything, we use the default D3 formatter.
+		 */
+		y2AxisFormatter: func,
+		/**
+		 * Stack the y2 axis data. This is only useful if you have multiple
+		 * `y2AxisFields`. Stacking will cause the chart to be aggregated by sum.
+		 */
+		y2AxisIsStacked: bool,
+		/**
+		 * Display points along with the y2 axis lines.
+		 */
+		y2AxisHasPoints: bool,
+		/**
+		 * There are some cases where you need to only show a "sampling" of ticks
+		 * on the y2 axis. This number will control that.
+		 */
+		y2AxisTickCount: number,
+		/**
+		 * Set a title for the y2 axis.
+		 */
+		y2AxisTitle: string,
+		/**
+		 * Set a color for the y2 axis title. This takes any number 0 or greater and
+		 * it converts it to a color in our color palette.
+		 */
+		y2AxisTitleColor: number,
+	},
+
+	getDefaultProps() {
+		return {
+			height: 400,
+			width: 1000,
+			margin: {
+				top: 10,
+				right: 80,
+				bottom: 65,
+				left: 80,
+			},
+
+			xAxisField: 'x',
+			xAxisTickCount: null,
+			xAxisTitle: null,
+			xAxisTitleColor: -1,
+
+			yAxisFields: ['y'],
+			yAxisIsStacked: false,
+			yAxisMin: 0,
+			yAxisHasPoints: true,
+			yAxisTickCount: null,
+			yAxisTitle: null,
+			yAxisTitleColor: -1,
+
+			y2AxisFields: null,
+			y2AxisIsStacked: false,
+			y2AxisHasPoints: true,
+			y2AxisMin: 0,
+			y2AxisTickCount: null,
+			y2AxisTitle: null,
+			y2AxisTitleColor: -1,
+		};
+	},
+
+	render() {
+		const {
+			className,
+			height,
+			width,
+			margin,
+			data,
+
+			xAxisField,
+			xAxisTickCount,
+			xAxisTitle,
+			xAxisTitleColor,
+			xAxisFormatter = formatDate,
+			xAxisMin = minByFields(data, xAxisField),
+			xAxisMax = maxByFields(data, xAxisField),
+
+			yAxisFields,
+			yAxisFormatter,
+			yAxisHasPoints,
+			yAxisIsStacked,
+			yAxisTickCount,
+			yAxisTitle,
+			yAxisTitleColor,
+			yAxisMin,
+			yAxisMax = yAxisIsStacked
+				? maxByFieldsStacked(data, yAxisFields)
+				: maxByFields(data, yAxisFields),
+
+			y2AxisFields,
+			y2AxisFormatter,
+			y2AxisHasPoints,
+			y2AxisIsStacked,
+			y2AxisTickCount,
+			y2AxisTitle,
+			y2AxisTitleColor,
+			y2AxisMin,
+			y2AxisMax = y2AxisFields && y2AxisIsStacked
+				? maxByFieldsStacked(data, y2AxisFields)
+				: maxByFields(data, y2AxisFields),
+
+			...passThroughs,
+		} = this.props;
+
+		// TODO: Consider displaying something specific when there is no data,
+		// perhaps a loading indicator.
```

Comment:
Yeah I'll just render a blank svg for now so the page doesn't reflow when the data finally comes in